configuration/backend_configuration/ampathformstranslations/art_therapy_form_translations_fr.json

```diff
@@ -1,11 +1,12 @@
 {
   "uuid": "",
   "form": "ART Therapy Form",
-  "Use this form to: start ART, substitute an ART line, switch an ART regimen, to stop ART or to restart ART": "Utilisez ce formulaire pour : commencer un traitement ARV, remplacer une ligne ARV, changer de régime ARV, arrêter un traitement ARV ou recommencer un traitement ARV.",
   "description": "Traductions en français pour 'ART Therapy Form'",
   "language": "fr",
   "translations": {
-    "Introduction": "Introduction",
+    "ART Therapy Form": "Formulaire de thérapie ART",
```

Comment:
@VeronicaMuthee this is not the way to add the translation for the form name, please refer the [iniz documentation](https://github.com/mekomsolutions/openmrs-module-initializer/blob/main/readme/ampathformstranslations.md) that I linked in the previous PR as well, you have to add the attribute `form_name_translation` and it should be not inside the `translations`